apps/omg_db/test/omg_db/recorder_test.exs

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 OmiseGO Pte Ltd
```

Comment:
re this file's location and the others moved: 

per convention it should be
```
apps/omg_db/test/application_test.exs
apps/omg_db/test/db_test.exs
apps/omg_db/test/recorder_test.exs
```

I think I can recall your proposition for this move (`mix test test/omg_db`, wasn't it?), but maybe it's better to discuss the new convention and potentially adopt in a separate PR? 

Also, isn't there another way to do this kind of test selection :thinking: ?

Reply:
I am not aware how one can run only tests specific to an application. But yeah. I’ve moved the tests because I wanted to run db tests only locally 😁

Reply:
Looking at the phoenix framework tets location, https://github.com/phoenixframework/phoenix/tree/master/test, they’re located under another subfolder-app name.